addons/point_of_sale/i18n/point_of_sale.pot

```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 16.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-12-15 12:48+0000\n"
-"PO-Revision-Date: 2022-12-15 12:48+0000\n"
+"POT-Creation-Date: 2023-01-03 15:25+0000\n"
+"PO-Revision-Date: 2023-01-03 15:25+0000\n"
```

Comment:
can you remove this change ? I know it's automatically done but apparently it's a rule to omit this when modifying the pot file